ddtrace/tracer/writer_test.go

```diff
@@ -275,6 +275,21 @@ func TestLogWriterOverflow(t *testing.T) {
 		assert.Equal(io.EOF, err)
 	})
 }
+func TestJSONEncodeSpanNewLines(t *testing.T) {
```

Comment:
Empty line above. Also, can't this fit in another test as a sub-test? Maybe inside `TestLogWriter` as a sub-test called `"escape"`? Does it need to be a separate test?